docs/tutorials/java/mxnet_java_on_intellij.md

```diff
@@ -108,6 +108,12 @@ TODO
 After clicking Finish, you will be presented with the project's first view.
 The project's `pom.xml` will be open for editing.
 
+**IntelliJ IDEA Alternative** If you want to use only Mave to create the project, you can create a new folder and run the following in the newly created folder :
+```bash
```

Comment:
please add further clarification here:
mkdir example
cd example/
then run the command

Reply:
Thanks for pointing it out. Updated it.